apps/web/src/server/lib/nunjucks-template-builders/summary-list-builder.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @param {any} header
+ * @param {any[]} rowArray
+ * @returns {object}
+ */
+export function generateSummaryList(header, rowArray) {
```

Comment:
Be nice to see a test unit for this, as it's one of the core bits of reusable functionality (and should be straight forward to test)